ci/jenkins/jobs/projects.yaml

```diff
@@ -217,19 +196,25 @@
             - e2e-lock-per-testbed
           throttle_concurrent_builds_enabled: 'true'
           status_context: jenkins-conformance
-          status_url: --none--
+          status_url: null
           success_status: Build finished.
           failure_status: Failed. Add comment /test-conformance to re-trigger.
           error_status: Failed. Add comment /test-conformance to re-trigger.
           triggered_status: null
           started_status: null
           wrappers: []
-          publishers: []
-          branches:
-          - ${{sha1}}
-          included_regions: []
-          cron: ''
-          ignore_post_commit_hooks: null
+          publishers:
+          - archive:
+              allow-empty: true
+              artifacts: '*sonobuoy*.tar.gz'
+              case-sensitive: true
+              default-excludes: true
+              fingerprint: false
+              only-if-success: false
+          - trigger-parameterized-builds:
```

Comment:
I'm glad that "trigger-parameterized-builds" worked. So whether the job is successful or not, it'll cleanup the workload cluster. I believe you have tested the test fail and cleanup scenario?

Reply:
I manually created a workload cluster and it got cleaned after the job run.

Reply:
Yes, we have a failed build and cleanup job started after that.